mingw-w64-libnpupnp/PKGBUILD

```
@@ -0,0 +1,50 @@
+# Contributor: Rosen Penev <rosenp@gmail.com>
+
+_realname=libnpupnp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=6.0.5
+pkgrel=1
+pkgdesc="A C++ base UPnP library, derived from Portable UPnP, a.k.a libupnp"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang32' 'clang64' 'clangarm64')
```

```suggestion
mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
```

No more 32 bit packages https://www.msys2.org/news/#2023-12-13-starting-to-drop-some-32-bit-packages

---

Oh that's unfortunate. Makes sense though.